docs/products/redis/troubleshooting/troubleshoot-redis-connection-issues.rst

```diff
@@ -0,0 +1,21 @@
+Troubleshoot Redis®* connection issues
+######################################
+
+Discover troubleshooting techniques for your Redis®* service and check out how to resolve common connection issues.
+
+Important notes
+---------------
+By default :doc:`Aiven for Redis® uses SSL connecions</docs/products/redis/howto/manage-ssl-connectivity>` and as such, these connections are closed automatically after 12 hours. This is not a configurable parameter so it cannot be changed. Aiven also sets a default value of 300 seconds for the redis_timeout advanced parameter.
```

Comment:
Is that `connecions` a typo? Or part of the link?

Reply:
Good catch, I thought I fixed this up previously :grimacing: thank you :grin: